deployment/custom-deployment/lib/cdk-textract-stack.ts

```diff
@@ -163,29 +169,37 @@ export class CdkTextractStack extends cdk.Stack {
       cloudfrontDocumentsBucketPolicyStatement
     );
 
+    
+    const esEncryptionKey = new kms.Key(this, 'esEncryptionKey', {
+      enableKeyRotation: true
+    });
     // Elasticsearch
     const elasticSearch = new es.CfnDomain(
       this,
       this.resourceName("ElasticSearchCluster"),
       {
         elasticsearchVersion: "6.5",
         elasticsearchClusterConfig: {
-          instanceType: "t2.medium.elasticsearch"
```

Comment:
Is there a specific reason for this? I thought Curtis deliberately reduced this size to reduce cost, and said that there was no performance loss.

Reply:
For security reasons, we had to switch back the instance Type
T2 class does not support encryption at rest. Supported Instance types can be found [here](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/aes-supported-instance-types.html)

Reply:
Let's make this m5.large.elasticsearch (still twice the cost of t2.medium, but half the cost of the m5.xlarge).

Reply:
Done!